lib/straight/blockchain_adapters_dispatcher.rb

```diff
@@ -34,8 +34,12 @@ def execute_in_parallel(block, adapters)
       attempts_counter = Concurrent::MVar.new(adapters_to_run.size)
       pool             = Concurrent::ThreadPoolExecutor.new
       adapters_to_run.each do |adapter|
-        p = Concurrent::Promise.new(executor: pool) { block.call(adapter) }
+        p = Concurrent::Promise.new(executor: pool) {
```

Comment:
May be it's good idea, but how many people read this article? How many people will ask you: why do you use this `{ .. }` for multiline code? We have the ruby style guide https://github.com/bbatsov/ruby-style-guide#user-content-single-line-blocks, let's use it.